src/main/java/br/com/caelum/vraptor/util/jpa/extra/ParameterLoader.java

```diff
@@ -69,32 +73,29 @@ public ParameterLoader(EntityManager em, HttpServletRequest request, ParameterNa
 		this.flash = flash;
 	}
 
+	@Accepts
 	public boolean containsLoadAnnotation(ControllerMethod method) {
```

Comment:
So, it's better that I put an if that always checks if the `ControllerMethod` has the `@Load` annotation in any parameter?
I think it kind of breaks the way that interceptors are build. By the way, does this version of VRaptor has the `@Lazy` annotation? Then any dependencies would be injected unnecessarily.